Artifacts/windows-chocolatey/install-choco-package.ps1

```diff
@@ -8,6 +8,12 @@ param(
     # Space-, comma- or semicolon-separated list of Chocolatey packages.
     [string] $Packages,
 
+    # Boolean indicating if we should allow empty checksums. Default to true to match previous artifact functionality despite security
+    [bool] $AllowEmptyChecksums = $true,

```

Comment:
Don't change this. Just pointing out that `bool` is fine. However, we can also use `switch`. In the `artifactfile.json` you would do the same as you're doing now, but instead of a space you would use a colon. For example,
```json
''' -AllowEmptyChecksums:$', parameters('allowEmptyChecksums'), '\"')]"
```